benchmark/napi/function_args/index.js

```diff
@@ -10,14 +10,14 @@ let v8;
 let napi;
 
 try {
-  v8 = require('./build/Release/binding');
+  v8 = require(`./build/${common.buildType}/binding`);
```

Comment:
Isn't this a boolean?

Reply:
`common.buildType` is a string:

```console
> require('./test/common').buildType
'Release'
>
```

Reply:
But this is `./benchmark/common`? (see above)

Reply:
Ah, whoops! Yes, of course, you are correct!